core/blockchain.go

```diff
@@ -1188,8 +1188,15 @@ func (bc *BlockChain) writeKnownBlock(block *types.Block) error {
 	return nil
 }
 
+// EvmTxTraces groups trace data from each executation of tx and left
+// some field to be finished from blockResult
+type EvmTxTraces struct {
```

Comment:
Moving `EvmTxTraces` to `core/types/l2trace.go` might be more concentrated.

Reply:
get it, may us just move to https://github.com/scroll-tech/go-ethereum/pull/101 for more reviewing?